Source/WebKit/Shared/LogStream.cpp

```diff
@@ -89,12 +92,17 @@ void LogStream::setup(uint64_t pid, IPC::StreamServerConnectionHandle&& serverCo
     m_logStreamConnection = IPC::StreamServerConnection::tryCreate(WTFMove(serverConnection), { });
     m_logWorkQueue = IPC::StreamConnectionWorkQueue::create("Log work queue"_s);
     if (RefPtr logStreamConnection = m_logStreamConnection) {
-        logStreamConnection->open(*m_logWorkQueue);
+        if (RefPtr logWorkQueue = m_logWorkQueue)
```

Comment:
All these if (workQueue) codes are incorrect, we cannot be in the code and not have the work queue, it makes no sense.

UI process cannot have a stream work queue per WCP. The queue should be a singleton.